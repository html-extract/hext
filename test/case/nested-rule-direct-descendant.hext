<div id>
  {
    <div class:class>
      {
        <p @text:p />
      }
    </div>
  }
</div>
