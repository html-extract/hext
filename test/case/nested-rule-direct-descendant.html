<html>
  <body>
    <div id="list1" class="list1">
      <div class="item1"><p>One</p>  </div>
      <div class="item2"><p>Two</p>  </div>
      <div class="item3"><p>Three</p></div>
    </div>
    <div id="list2" class="list2">
      <div class="item4"><p>Four</p> </div>
      <div class="item5"><p>Five</p> </div>
      <div class="item6"><p>Six</p>  </div>
    </div>
  </body>
</html>
