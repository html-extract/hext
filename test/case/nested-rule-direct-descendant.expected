{"class":["item1","item2","item3"],"p":["One","Two","Three"]}
{"class":["item4","item5","item6"],"p":["Four","Five","Six"]}
